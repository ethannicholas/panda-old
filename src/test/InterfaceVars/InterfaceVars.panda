interface Broken {
    var test := 0
    property test2 := 1
    def test3 := 2
}