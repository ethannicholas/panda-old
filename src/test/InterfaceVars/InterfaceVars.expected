error: InterfaceVars.panda:2:5: interfaces may not contain fields
error: InterfaceVars.panda:3:5: interfaces may not contain fields
error: InterfaceVars.panda:4:5: interfaces may not contain fields
3 errors
